test/bundler/bundler_compile.test.ts

```diff
@@ -263,17 +263,23 @@ describe("bundler", () => {
     run: { stdout: "ok" },
   });
 
-  for (const additionalOptions of [
+  const additionalOptionsIters: Array<{
+    bytecode?: boolean;
+    minify?: boolean;
+    format: "cjs" | "esm";
+  }> = [
     { bytecode: true, minify: true, format: "cjs" },
     { format: "cjs" },
     { format: "cjs", minify: true },
     { format: "esm" },
     { format: "esm", minify: true },
-  ]) {
+  ];
+
+  for (const additionalOptions of additionalOptionsIters) {
     const { bytecode = false, format, minify = false } = additionalOptions;
     const NODE_ENV = minify ? "'production'" : undefined;
     itBundled("compile/ReactSSR" + (bytecode ? "+bytecode" : "") + "+" + format + (minify ? "+minify" : ""), {
-      install: ["react@next", "react-dom@next"],
+      install: ["react@19.2.0-canary-b94603b9-20250513", "react-dom@19.2.0-canary-b94603b9-20250513"],
```

Comment:
it would be nice if `itBundled` supported lockfiles so we could be sure transitive dependencies don't change